_posts/2023-02-14-bio-formats-6-12-0.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Release of Bio-Formats 6.12.0
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 6.12.0
+---
+
+Today we are releasing Bio-Formats 6.12.0 with a number of significant changes. Most notably for the developer community you will find that Bio-Formats has now been updated to fully support Java 17.
+
+Additionally in this release there have been changes to the logging dependencies which will require action from developers who consume Bio-Formats. Bio-Formats previously included a logback dependency which is now marked as optional, this will require developers to directly declare a dependency for a SLF4J binding such as [logback](https://logback.qos.ch/). For further details on the changes to the logging dependencies and links to examples please see the [logging documentation](https://docs.openmicroscopy.org/bio-formats/latest/developers/logging.html) page.
```

Comment:
```suggestion
Additionally in this release there have been changes to the logging dependencies which will require action from developers who consume Bio-Formats. Bio-Formats previously included a logback dependency which is now marked as optional, this will require developers to directly declare a dependency for a SLF4J binding such as [logback](https://logback.qos.ch/). For further details on the changes to the logging dependencies and links to examples please see the [logging documentation](https://bio-formats.readthedocs.io/en/stable/developers/logging.html) page.
```